src/cross_module_fn.h

```diff
@@ -99,6 +99,7 @@ typedef struct CrossModuleFunctions
 	bool (*process_compress_table)(AlterTableCmd *cmd, Hypertable *ht,
 								   WithClauseResult *with_clause_options);
 	void (*process_altertable_cmd)(Hypertable *ht, const AlterTableCmd *cmd);
+	void (*process_rename_cmd)(Hypertable *ht, const RenameStmt *stmt);
```

Comment:
I think this has come up before, and I don't know the conclusion so I'll reiterate.

I think creating a new cross-module function for every process utility (DDL) command is not a good idea and doesn't really scale. We already "forward" DDL commands to TSL and we should try to use that mechanism or improve it if it isn't sufficient in its current form.

The next best approach, IMO, is to have a "general" cross-module function that we can reuse for all DDL commands, e.g.:

```
void (*process_ddl_cmd)(Hypertable *ht, const Node *cmd);
```

then use a switch on it in the `process_ddl_cmd` function:

```

switch (nodeTag(cmd))
{
    case T_RenameStmt:
         ....
         break;
    case T_AlterTableCmd:
         ...
         break;
}
```